Formula/p/pypy3.10.rb

```diff
@@ -72,6 +72,22 @@ def install
       s.gsub! "/include'", "/include/tcl-tk'"
     end
 
+    if OS.mac?
+      # Allow python modules to use ctypes.find_library to find homebrew's stuff
+      # even if homebrew is not a /usr/local/lib. Try this with:
+      # `brew install enchant && pip install pyenchant`
+      inreplace "lib-python/3/ctypes/macholib/dyld.py" do |f|
+        f.gsub! "DEFAULT_LIBRARY_FALLBACK = [",
+                "DEFAULT_LIBRARY_FALLBACK = [ '#{HOMEBREW_PREFIX}/lib', "
+        f.gsub! "DEFAULT_FRAMEWORK_FALLBACK = [", "DEFAULT_FRAMEWORK_FALLBACK = [ '#{HOMEBREW_PREFIX}/Frameworks',"
+      end
+
+      # Fix mismatch in test definition of ncurses
+      inreplace "lib_pypy/_curses_build.py" do |f|
+        f.gsub! "char * tparm(const char *, ...);", "char * tparm(char *, ...);"
+      end
```

Comment:
Can we apply this as a patch (https://github.com/pypy/pypy/commit/13e2004d86a3431d27a2ac98c7d591473ca9ef9c.patch?full_index=1) instead? Makes it much more obvious it's something that will be fixed in a future release rather than being some hack

Reply:
That seems like a good idea. When I get a minute I'll make that mod.

Reply:
done